src/AspNetCore/Src/managedexports.cxx

```diff
@@ -158,19 +159,22 @@ http_read_request_bytes(
     _Out_ BOOL* pfCompletionPending
 )
 {
-    HRESULT hr;
+    HRESULT hr S_OK;
+    IN_PROCESS_STORED_CONTEXT* pInProcessStoredContext = NULL;
+    IHttpRequest * pHttpRequest = NULL;
+    BOOL fAsync = TRUE;
 
     if (pHttpContext == NULL)
     {
         return E_FAIL;
     }
+
     if (dwCbBuffer == 0)
     {
         return E_FAIL;
     }
-    IHttpRequest *pHttpRequest = (IHttpRequest*)pHttpContext->GetRequest();
```

Comment:
What's up with these changes? They look unrelated?

Reply:
This is just style in keeping the declarations at the beginning of the file.